src/ruby/spec/generic/rpc_server_pool_spec.rb

```diff
@@ -114,17 +130,17 @@
 
   describe '#start' do
     it 'runs pre-scheduled jobs' do
```

Comment:
But this test is no longer about pre-scheduled jobs, correct? In fact, the concept is now meaningless (and maybe always was). How about just dropping this test or changing it to something else